__tests__/user-details/Intro-button.test.js

```diff
@@ -1,6 +1,8 @@
 const puppeteer = require('puppeteer');
 const { userDetails } = require('../../mock-data/user-details/index');
 
+const API_BASE_URL = 'https://staging-api.realdevsquad.com';
```

Comment:
Please define it at one please and import it from there, instead of putting it in every file?